### ISSUE TITLE
Horizon on Azure sysprep issue

### ISSUE THREAD

**Issue by Retched1** [The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool#120]
I am trying to use this tool to optimize a Win 10 21H2 MultiSession VM used by Horizon on Azure. (In place of the VMware OSOT tool) When I have my image loaded with apps, and am logged on as a local/only Admin user, I run the ps1 as I have done so in the past, reboot and all is well. 
When I go to convert the image to a published image that can be used (Sysprep is run at this step by the Horizon solution on Azure), the Vms all get stuck after 20 minutes with an error ""Power off virtual machine 'TEST01' - Timeout Error Waited 20 minutes for virtual machine 'TEST01' to power off.""
The local Sysprep log in c:\Panther doesn't give me much to go on.
I'm curious if anyone else is using this script with Horizon on Azure, and if they have any pointers on this error. VM and Msft seem to point to issues with apps being removed from the profile being used to syprep
https://docs.microsoft.com/en-us/troubleshoot/windows-client/deployment/sysprep-fails-remove-or-update-store-apps

I do not think the script is broken, just not sure where else to ask this honestly.

**Comment by Retched1** [comment]
So I created an Admin2 account and made it a local admin.
Logged in as Admin2, and deleted the Administrator profile.
I then ran the VDOT as this Admin2 and rebooted and logged in as the original administrator.
I then publish the image and used the Administrator account credentials to perform the Sysprep as part of the process, and it seemed to work.
I believe the article above was correct, that a previously created account (prior to removal of appx packages) caused the issue for me in my case.
